test/e2e/dynamic_provisioning_test.go

```diff
@@ -262,6 +262,8 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 	})
 
 	ginkgo.It("should create a deployment object, write and read to it, delete the pod and write and read to it again [kubernetes.io/azure-disk] [disk.csi.azure.com] [Windows]", func() {
```

Comment:
Add a TODO for skipped tests? it should be added back sometime later when the issue fixed.

Reply:
there is already an issue(https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues/381) to track this

Reply:
could you approve? thanks.